docs/conf.py

```diff
@@ -257,3 +257,10 @@
     import_module('camb')
 except ImportError:
     autodoc_mock_imports.append('camb')
+
+# Sphinx Gallery
+extensions += ['sphinx_gallery.gen_gallery',]
```

Comment:
Codeclimate seems to complain about this
```suggestion
extensions += ['sphinx_gallery.gen_gallery', ]
```